lib/apollo.ts

```diff
@@ -0,0 +1,8 @@
+import { ApolloClient, InMemoryCache } from "@apollo/client";
+
+const apolloClient = new ApolloClient({
+  uri: "http://localhost:3000/api/graphql",
```

Comment:
We will need to replace the hard-coded URL with an environment variable. But, it is good to do in the next sprint.
I will make a ticket for the small task on Jira.